src/ui/components/AppWrapper/AppWrapper.tsx

```diff
@@ -430,6 +430,10 @@ const AppWrapper = (props: { children: ReactNode }) => {
     // @TODO - foconnor: This is a temp hack for development to be removed pre-release.
     // These items are removed from the secure storage on re-install to re-test the on-boarding for iOS devices.
     try {
+      if (process.env.DEV_SKIP_ONBOARDING === "true") {
```

Comment:
Why is this in the same try block as `APP_ALREADY_INIT`?

Reply:
Sorry, I thought the call before app-already-init was in that try block, we just need to call before APP_ALREADY_INIT in initApp, right?

Reply:
Yeah just move this if statement above the try block (and the todo comment)